src/containers/Header.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { unSelect } from '../actions/index';
+import logo from '../icons/main.png';
+import '../css/header.css';
+
+class Logo extends Component {
+  render() {
+    return (
+      <div className='header__wrapper'>
+        <img src={logo} onClick={() => this.props.unSelect()} className='header__logo' alt='logo'/>
```

Comment:
you don't need to pass function which returns another function `() => this.props.unSelect()`. Just use `this.props.unSelect`.

Small note: change `unSelect` with `deSelect`